src/components/LoadingSmall.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Loader, Segment } from 'semantic-ui-react'
+import { useLanguageProvider } from '../contexts/Localisation'
+
+const LoadingSmall: React.FC = () => {
+  const { strings } = useLanguageProvider()
+  return (
+    <Segment basic style={{ height: '50%', margin: 0}}>
+      <Loader active size="mini">
+        {/* {strings.LABEL_LOADING} */}
+      </Loader>
```

Comment:
Without the inner string, we can just make this a self-closing tag:
```
<Loader active size="small" />
```

Also -- I think "small" is better in this case, when compared with the block that appears. However, I can see that "mini" or "tiny" will probably be better when you get to the "Assignment" page lines, so why not make it a prop with a default.

So all up component would be something like:

```ts
const LoadingSmall: React.FC<{ size?: SemanticSIZES }> = ({ size = 'small' }) => {
  return (
    <Segment basic>
      <Loader active size={size} />
    </Segment>
  )
}
```